packages/runtime/src/next/components/tests/controls/rich-text-v2-control/__snapshots__/client.test.tsx.snap

```diff
@@ -1,6 +1,6 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`Page RichTextV2 renders a makeswift::controls::rich-text-v2::mode::block placeholder when empty: resolvedValue 1`] = `
+exports[`renders a makeswift::controls::rich-text-v2::mode::block placeholder when empty: resolvedValue 1`] = `
```

Comment:
Cleaned up/simplified existing snapshots' titles. No changes to snapshot values.